### PR TITLE
Add passwdstatus hostinfo plugin

### PR DESCRIPTION
**Pull request by ryandub**
Report passwd info for users:
* status
* last changed
* minimum age
* warning period
* inactivity period

### CODE REVIEW

File: hostinfo/passwdstatus.lua [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    return
+  end
+
+  function collectData(users)
+    --[[ Asynchronous gather user passwd info ]]--
+    local obj = {}
+
+    forEachLimit(users, 5, function(user, callback)
+      local child
+      child = spawn('passwd', {'-S', user}, {})
```
Comment (pquerna) [inline hostinfo/passwdstatus.lua]: wish we just had a helper for this pattern -- its becoming extremely commony, its basically spawn+captureAll

Comment (rphillips) [inline hostinfo/passwdstatus.lua]: we do have process.execFile : https://github.com/luvit/luvit/blob/master/lib/luvit/childprocess.lua#L44

Comment (ryandub) [inline hostinfo/passwdstatus.lua]: @rphillips could you elaborate how `execFile` would be used in this case?

Comment (rjemanuele) [inline hostinfo/passwdstatus.lua]: execFile *may* be useful there.  It lacks the stream end functionality which ensures the all the stream is read as opposed to assuming the stream is fully read on proc exit.  Fix it or create a new version like this?  https://gist.github.com/rjemanuele/4ac7473464b74b9a721d

**Comment by kaustavha** [comment]
Duplicate of #779

**Pull request closed without merging**